### PR TITLE
Bug fixes in UI: ruleengine, pipeline summary + UI down modal messaging

### PR DESCRIPTION
**Pull request by ajainarayanan**
[CDAP-12417](https://issues.cask.co/browse/CDAP-12417) - Fixes subtitle while showing UI down. Changes based on environment.
[CDAP-12393](https://issues.cask.co/browse/CDAP-12393) - Hide Average run time for realtime pipeline
[CDAP-12433](https://issues.cask.co/browse/CDAP-12433) - Fixes UI to update rules tab when importing a rulebook.

### CODE REVIEW

File: cdap-ui/app/cdap/services/helpers.js [hunk 1]
```diff
@@ -257,6 +257,10 @@ const isPluginSource = (pluginType) => {
   return ['batchsource', 'realtimesource', 'streamingsource'].indexOf(pluginType) !== -1;
 };
 
+const isBatchPipeline = (pipelineType) => {
+  return ['cdap-etl-batch', 'cdap-data-pipeline'].indexOf(pipelineType) !== -1;
```
Comment (elfenheart) [inline cdap-ui/app/cdap/services/helpers.js]: still need `cdap-etl-batch`?

File: cdap-ui/app/cdap/components/PipelineSummary/index.js [hunk 2]
```diff
@@ -130,7 +130,11 @@ export default class PipelineSummary extends Component {
     }
     Mousetrap.unbind('esc');
   }
-  componentDidMount() {
+
+  fetchStats = () => {
+    if (isBatchPipeline(this.props.pipelineType)) {
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/PipelineSummary/index.js]: shouldn't this be `if (!...)`? We do want to fetch stats if it is batch pipeline right?

**Review by elfenheart — APPROVED** [review]
👍  LGTM

**Comment by leacd** [comment]
LGTM

**Pull request merged**